### ISSUE TITLE
`space-in-parens` error for code in a comment block

### ISSUE THREAD

**Issue by necolas** [babel/babel-eslint#124]
This looks like a bug because it's reporting an error on a commented-out line.

Error:

```
  4:6  error  There should be no spaces inside this paren  space-in-parens
```

Source:

```js
 render() {
    // return (
    //   <div />
    // ); // <-- this is the line that is reported
  }
```

**Comment by hzoo** [comment]
@necolas Looks like the acorn/espree ast is different regarding comments, although not sure why that affects that rule.. - will investigate.

**Comment by necolas** [comment]
Thanks. And FYI, switching to a wrapping `/* */` comment avoids the false positive.

**Comment by sebmck** [comment]
@hzoo Try filtering out `CommentBlock` and `CommentLine` tokens.

**Comment by hzoo** [comment]
Kind of confused - and it looks like if it's not empty it also works?
```js
 render() {
    1;
    // return (
    //   <div />
    // ); // <-- this is the line that is reported
  }
```
@sebmck Sorry how would I do that? It looks like comments and tokens are in separate arrays? Sounds like a similar issue to https://github.com/babel/babel/issues/672 of using estraverse to attach comments to empty block `estraverse.attachComments(ast, comments, tokens);`? (Although this works with regular eslint so maybe not)?

**Comment by hzoo** [comment]
Ok so this is related to https://github.com/babel/babel-eslint/pull/76#issuecomment-95360914? I guess comments aren't attached in an empty block, which could be solved but not using estraverse like the `Attach comments in parser` task in https://github.com/babel/babel/issues/1486

**Comment by hzoo** [comment]
I guess this can be closed as a upstream babel issue actually

**Comment by hzoo** [comment]
Ok looks like this was also fixed in the other comment changes https://github.com/babel/acorn-to-esprima/commit/5da46d46e842897f5e3b261303994273e0933064

**Comment by hzoo** [comment]
@necolas let me know if it's still an issue
